### ISSUE TITLE
Incomplete Documentation: Flashing Firmware: Preparing firmware and a Python program

### ISSUE THREAD

**Issue by kjcole** [bbcmicrobit/micropython#707]
The section on  "Prepariing firmware and a Python program" in /docs/devguide/flashfirmware.rst consists of:

```
    Preparing firmware and a Python program
    ---------------------------------------
    
    tools/makecombined
     
    hexlify
```

which seems woefully incomplete.

**Comment by microbit-carlos** [comment]
Hi @kjcole,
Thanks for pointing this out, we are updating this part of the docs in PR https://github.com/bbcmicrobit/micropython/pull/711 if you'd like to review the changes as well.

**Comment by dpgeorge** [comment]
Should be addressed by e7bdb46f91b87304ea053fd69cceeb41a7623c9c
